scripts/XLSXHandler.py

```diff
@@ -232,6 +232,9 @@ def __handle_x_condition_block(self, block, domain, label):
             node_condition = re.sub(Dialog.X_PLACEHOLDER, row[0], block_header)
             node_name = self._dialogData.createUniqueNodeName(node_condition)  # make it unique
             nodeData = self._dialogData.createNode(node_name, domain)  # create space for new node, remembers node_name
+            if nodeData is None:
```

Comment:
I would move this error into the `createNode` function.

Reply:
this is now the same for createNode, createEntity and createIntent - it wold requre change in all the places
Are there any arguments to make the change? The general push is usually to log in higher levels  as there might be reason in the future to mask the error (though I am not sure at the moment why). Let's discuss.

Reply:
After discussion: move just info print, leave `exit()` here.